src/constants.zig

```diff
@@ -42,6 +42,9 @@ pub const vsr_checkpoint_ops = journal_slot_count -
     lsm_compaction_ops -
     lsm_compaction_ops * stdx.div_ceil(pipeline_prepare_queue_max * 2, lsm_compaction_ops);
 
+pub const vsr_repair_message_budget_max = replicas_max;
```

Comment:
On the fence on whether this should be replicas_max or replicas_max * 2

Reply:
Or `(replicas_max - 1) * 2` since we aren't sending to ourselves?

Reply:
repair timeout is 750, now this is one repair message every 250 ms, which actually feels too low, will bump to `*2`